common/src/models/resource.rs

```diff
@@ -74,19 +78,22 @@ fn get_databases_table(databases: &Vec<&Response>) -> String {
     }
 
     format!(
-        r#"These databases are linked to this service
+        r#"These databases are linked to {}
 {table}
 "#,
+        service_name.bold()
```

Comment:
I was thinking more of making "databases" bold  and not the service name. Not sure how you feel?

Reply:
Hmm, for me it's 50/50, so if you have a preference then we can go with that. :+1:

Reply:
Implemented now and it does make it clearer.